src/world/message_server.cpp

```diff
@@ -351,15 +361,12 @@ void message_server_listen(bool const& requestExit)
         // 2: zmq::message_t extra
         // 3: zmq::message_t packet
         message_server_parse((MSGSERVTYPE)ref<uint8>((uint8*)msgs[1].data(), 0), &msgs[2], &msgs[3], &msgs[0]);
-
-        // TODO: Is this still needed?
-        // sql_->TryPing();
     }
 }
 
 void cache_zone_settings()
 {
-    auto rset = db::query("SELECT zoneid, zoneip, zoneport, misc FROM zone_settings");
+    auto rset = db::preparedStmt("SELECT zoneid, zoneip, zoneport, misc FROM zone_settings");
```

Comment:
Since there are no params here, you should be fine to keep using query (unless the idea is to future proof against someone parameterizing this later)

Reply:
Prepared statements are faster because the database engine knows ahead of time about the query, so there is actually a beneficial change here

Reply:
For a query like this that has no params - sure, it doesn't really matter. As a general rule - the MariaDB C and C++ drivers aren't as battle-tested as we'd like, and a user could misconfigure their MariaDB service in any number of wacky ways, so I'd rather enforce our usage patterns through the code we control.

I'd like to eventually get to a point where there's only a single entry point to the db: `preparedStmt`. It allows us to do fancy things like make the call `constexpr` and register all the statements on startup etc.

Also performance and security, etc.

Reply:
That all makes sense. The main gain here being defense against injections for everything else, sticking to one entry point for all calls is a reasonable default. 

Will continue to lurk and stop interrupting y'all.